gulpfile.js/tasks/default.js

```diff
@@ -1,14 +1,12 @@
 'use strict'
 
-var gulp = require('gulp')
-var runSequence = require('run-sequence')
+const gulp = require('gulp')
+const runSequence = require('run-sequence')
 
-gulp.task('default', ['clean'], function () {
+gulp.task('default', (done) => {
   runSequence(
-    ['lint:gulpTasks', 'sass', 'images', 'svg', 'concatEncryption', 'error-pages'],
-    'test',
-    'modernizr',
     'pattern-library',
```

Comment:
It's not clear from this file which build gets run. If we added `build:v4` before `pattern-library` it shouldn't run twice right?